SwrveSDK/src/main/assets/youtubeapi.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<body style = "padding: 0; margin: 0;">
+<div style = "width: 100%; height: 100%;" id="player"></div>
+
+<script>
```

Comment:
Hmmm. The JS is loaded via web. So the latest Iframe API playe JS is whats used. Only the HTML is hardcoded.

There are certain things we want local and certain things we want remote. We'll just have to weigh the pros/cons of both and figure out which one we want to associate this with.